KomaMRICore/src/rawdata/ISMRMRD.jl

```diff
@@ -139,16 +130,16 @@ function signal_to_raw_data(
         "reconSize"                      => [Nx+Nx%2, Ny+Ny%2, 1],              #reconSpace>matrixSize
         "reconFOV"                       => Float32.([FOVx, FOVy, 1e-3]*1e3),   #reconSpace>fieldOfView_mm
         #encodingLimits
-        "enc_lim_kspace_encoding_step_0" => Limit(0, Nx-1, ceil(Int, Nx / 2)),  #min, max, center, e.g. phase encoding line number
-        "enc_lim_kspace_encoding_step_1" => Limit(0, Ny-1, ceil(Int, Ny / 2)),  #min, max, center, e.g. partition encoding number
-        "enc_lim_kspace_encoding_step_2" => Limit(0, 0, 0),                     #min, max, center, e.g. partition encoding number
-        "enc_lim_average"                => Limit(0, 0, 0),                     #min, max, center, e.g. signal average number
-        "enc_lim_slice"                  => Limit(0, 0, 0),                     #min, max, center, e.g. imaging slice number
-        "enc_lim_contrast"               => Limit(0, 0, 0),                     #min, max, center, e.g. echo number in multi-echo
-        "enc_lim_phase"                  => Limit(0, 0, 0),                     #min, max, center, e.g. cardiac phase number
-        "enc_lim_repetition"             => Limit(0, 0, 0),                     #min, max, center, e.g. dynamic number for dynamic scanning
-        "enc_lim_set"                    => Limit(0, 0, 0),                     #min, max, center, e.g. flow encoding set
-        "enc_lim_segment"                => Limit(0, 0, 0),                     #min, max, center, e.g. segment number for segmented acquisition
+        "enc_lim_kspace_encoding_step_0" => KomaMRIBase.Limit(0, Nx-1, ceil(Int, Nx / 2)),  #min, max, center, e.g. phase encoding line number
+        "enc_lim_kspace_encoding_step_1" => KomaMRIBase.Limit(0, Ny-1, ceil(Int, Ny / 2)),  #min, max, center, e.g. partition encoding number
+        "enc_lim_kspace_encoding_step_2" => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. partition encoding number
+        "enc_lim_average"                => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. signal average number
+        "enc_lim_slice"                  => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. imaging slice number
+        "enc_lim_contrast"               => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. echo number in multi-echo
+        "enc_lim_phase"                  => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. cardiac phase number
+        "enc_lim_repetition"             => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. dynamic number for dynamic scanning
+        "enc_lim_set"                    => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. flow encoding set
+        "enc_lim_segment"                => KomaMRIBase.Limit(0, 0, 0),                     #min, max, center, e.g. segment number for segmented acquisition
```

Comment:
Same as comment above

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/822fea1165c72a77178b98278ecf9ce4cfa55f96